File: src/Python/easy/math/problem1518.py
import math

class Solution:
    def numWaterBottles(self, numBottles: int, numExchange: int) -> int:
        sum = numBottles
        while numBottles >= numExchange:
            r = numBottles % numExchange
            b = math.floor(numBottles / numExchange)
            sum += b
            numBottles = b + r
        return sum

File: src/JavaScript/easy/math/problem1518.js
/**
 * @param {number} numBottles
 * @param {number} numExchange
 * @return {number}
 */
var numWaterBottles = function (numBottles, numExchange) {
    let sum = numBottles
    while (numBottles >= numExchange) {
        r = numBottles % numExchange
        b = Math.floor(numBottles / numExchange)
        sum += b
        numBottles = b + r
    }
    return sum
};
